### ISSUE TITLE
[Visual Requirement-Lifecycle Management] Elements must have sufficient color contrast (.mtk7).

### ISSUE THREAD

**Issue by sonimishra** [microsoft/monaco-editor#1757]
**User Experience:**
If, the Luminosity contrast ratio of the text is less than 4.5:1, then the low vision might get affected and does not able to read the text available on the page.

**Pre requisite:** 
1.Storage account should create for East US, West US location.
2. Some Rule should already add.

**Repro-Steps:**
1. Open Url: https://ms.portal.azure.com/
2. Login with valid credentials.
3. Navigate to left panel and select 'Storage Account' option.
4. Select an already created Storage account.
5. Navigate to "Lifecycle Management" option from left pane and select it.
6. Navigate to "Code View" tab and select it.
7. Navigate through all the controls appearing under "Code View" section.
8. Open Accessibility insights for web tool and select Fast pass option.
9. Verify, whether the element does have sufficient color contrast or not.

**Actual Result:**
On the Lifecycle Management page, elements does not have sufficient color contrast.

**Expected Result:**
On the Lifecycle Management page, elements should have sufficient color contrast, so that the low vision user can able to see properly.

**Note**: This issue also exist for all the Green color element available on the page.

**Issue**: Elements must have sufficient color contrast (color-contrast: https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI)

Target application title: testhclnewthree - Lifecycle Management - Microsoft Azure
Target application url: https://ms.portal.azure.com/#@microsoft.onmicrosoft.com/resource/subscriptions/936d12f6-dcd4-42f5-b8dc-5f7d89bfe78f/resourceGroups/accessibility/providers/Microsoft.Storage/storageAccounts/testhclnewthree/managementPolicies

Element path: .view-line:nth-child(11) > span > .mtk7

Snippet: <span class="mtk7">30</span>

**How to fix:**
Fix any of the following:
Element has insufficient color contrast of 4.48 (foreground color: #09885a, background color: #fffffe, font size: 10.5pt, font weight: normal). Expected contrast ratio of 4.5:1

This accessibility issue was found using Accessibility Insights for Web 2.5.5 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

Bug copied from Data life cycle : https://msazure.visualstudio.com/One/_workitems/edit/4848733
![MAS1 4 3-Luminosity contrast ratio](https://user-images.githubusercontent.com/30456910/71968934-c3e88700-322b-11ea-9618-30289c3810ee.png)
